docs/source/library/Visualization.rst

```diff
@@ -0,0 +1,27 @@
+Visualization
+=============
+
+.. currentmodule:: mqt.qmap
+
+All visualization functionality is bundled in the sub-module :mod:`mqt.qmap.visualization`.
+
+Search graph visualization
+##########################
+
+.. currentmodule:: mqt.qmap.visualization
+
+The recommended way to visualize search graphs of a mapping process is via a :class:`SearchVisualizer` object, which can be passed to the :func:`mqt.qmap.compile` function to enable data logging during mapping, after which this data can be visualized by calling the method :meth:`SearchVisualizer.visualize_search_graph`.
+
+.. note::
+    :meth:`SearchVisualizer.visualize_search_graph` returns an IPython display object and therefore requires to be executed in a jupyter notebook or jupyter lab environment.
+
+.. note::
+    Automatic layouting of architecture or search nodes requires `Graphviz <https://graphviz.org/>`_ to be installed (except for the layouting method :code:`walker`). If Graphviz is called without it being installed, it will ensue in an error such as:
+
+    :code:`FileNotFoundError: [Errno 2] "sfdp" not found in path.`
+
+    Consequently, the only way to use :meth:`SearchVisualizer.visualize_search_graph` without Graphviz is by passing explicit architecture node positions or hiding the architecture graph by passing :code:`show_layout=None`.
```

Comment:
Oh, great, I thought the package was just a wrapper and would still require a normal installation of Graphviz, but it seems to work without it. Thank you!

Reply:
Or maybe it actually is just a wrapper after all... At least, the CI sphinx build seems to fail with the "sfdp file or directory not found" error, that usually only occurs if Graphviz is not installed/not found. And also on https://pypi.org/project/graphviz/ it states, that for some tasks full Graphviz needs to be installed separately to the pip package.

Reply:
Ok. Just spent a couple of hours trying to get the docs build to run as expected. Unfortunately, without much success. I simply cannot build the documentation locally and render the interactive widget as part of the docs page (which should be supported as far as I found out). My main observation from inspecting browser logs was that it must be due to some incompatibility between MathJax 3 and whatever is being used for embedding Widgets.
In the end, I gave up. Maybe sometime in the future this will magically work. Until then, `graphviz` is at least installed in the RtD environment.